src/application/index.js

```diff
@@ -1,36 +1,59 @@
 import React from "react";
 import { render } from "react-dom";
-import { StorageContextProvider } from "./context/StorageContextProvider";
-import Panel from "./components/Panel";
-import { BridgeProvider } from "./components/bridge";
+import { ApolloClient, ApolloProvider, InMemoryCache, ApolloLink , useQuery, gql } from "@apollo/client";
+// import Panel from './components/Panel';
 
-const isChrome = typeof chrome !== "undefined" && !!chrome.devtools;
-const isDark = isChrome ? chrome.devtools.panels.themeName === "dark" : false;
+const cache = new InMemoryCache({
+  typePolicies: {
+    Query: {
+      fields: {
+        queries() {
+          return queriesVar();
+        },
+        mutations() {
+          return mutationsVar();
+        },
+        cache() {
+          return cacheVar();
+        }
+      }
+    }
+  }
+});
 
-export const initApp = shell => {
-  shell.connect(bridge => {
-    window.bridge = bridge;
-    if (isChrome) chrome.runtime.sendMessage("apollo-panel-load");
-    const app = (
-      <BridgeProvider bridge={bridge}>
-        <StorageContextProvider storage={shell.storage}>
-          <Panel
-            isChrome={isChrome}
-            bridge={bridge}
-            theme={isDark ? "dark" : "light"}
-          />
-        </StorageContextProvider>
-      </BridgeProvider>
-    );
+const queriesVar = cache.makeVar(null);
```

Comment:
Using reactive vars to store the data. At first, I used `writeQuery` but when querying for the `cache` object, I received a lot of errors. Guessing that there are some issues with returning a cache in a cache which makes sense. I still had a bit of trouble with the cache object here but JSON.stringifying it worked like a charm. 

I'm cool with how this is being handled here but if you see a potential issue or a better option, please let me know. 🙏

Reply:
This works, but it definitely feels a little bit "because we can"-ish. However, I don't immediately have a simpler suggestion, so let's go with this for now!